examples/scenarios/wefe_pv_panel/scripts/infer.py

```diff
@@ -26,4 +28,15 @@
     **kwargs,
 )
 
-# run building.infer, reopen json file and ammend datapackage
\ No newline at end of file
+# this function adds any additional foreign keys to the datapackage.json file
+otp_building.add_foreign_keys_to_datapackage(
+    "datapackage.json",
+    {
```

Comment:
This is a good first step, but I think we can infer the `from_bus` and `to_bus` bits automatically inside the `[rl-institut/oemof-tabular-plugins]` function I would say

Reply:
yes that's what I tried to do here: `"from_to_bus": ["pv-panel"]`
but for some reason it doesn't add to the pv-panel component